addon/components/menu.js

```diff
@@ -33,7 +33,7 @@ export default class Menu extends Component {
     if (this.isOpen) {
       this.isOpen = false;
       next(() => {
-        document.getElementById(this.buttonGuid).focus();
+        document.getElementById(this.buttonGuid)?.focus();
```

Comment:
I'm always a little confused about Babel, but does the optional chaining used here require the consuming app to have the Babel plugin (https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining.html) installed?

Is there any chance this could cause grief to app's who's users don't use a browser that yet supports optional chaining (https://caniuse.com/mdn-javascript_operators_optional_chaining) ?

Should we just go the long hard form here to be safe?

/cc @far-fetched @GavinJoyce

Reply:
I think it will break with old versions. I guess the question is how far we want to go with browser support.

Reply:
I think where this case is concerned, my preference is to go with old school `if`. Can’t see a lot to be gained using optional chaining here.